include/rogue/LibraryBase.h

```diff
@@ -89,7 +89,7 @@ namespace rogue {
          uint64_t getFieldUInt64(std::map<std::string, std::string> fields, std::string name);
```

Comment:
Should the other types (e.g. getFieldUint64) have a `bool def=false` argument?

Reply:
Maybe eventually as required. In reality this may just be a short term stop gap to avoid issues with .h files generated with the old API.